site/content/integrate/slash-commands/_index.md

```diff
@@ -17,7 +17,7 @@ Slash commands are messages that begin with `/` and trigger an HTTP request to a
 Slash commands have an additional feature, **autocomplete**, that displays a list of possible commands based on what has been typed in the message box. Typing `/` in an empty message box will display a list of all slash commands. As the slash command is typed in the message box, autocomplete will also display possible arguments and flags for the command.
 
 Unlike [outgoing webhooks]({{< ref "/integrate/webhooks/outgoing" >}}), slash commands work in private channels and direct messages in addition to public channels, and can be configured to auto-complete when typing.
-Mattermost includes a number of [built-in slash commands]({{< ref "built-in" >}}). You can also create [custom slash commands]({{< ref "custom" >}}).
+Mattermost includes a number of [built-in slash commands](https://docs.mattermost.com/channels/interact-with-channels.html). You can also create [custom slash commands]({{< ref "custom" >}}).
```

Comment:
Are we able to use a permalink for this? e.g. How we use permalink for this https://mattermost.com/pl/software-and-services-license-agreement, which forwards to https://mattermost.com/software-services-license-agreement. We used to have a lot of links like https://mattermost.com/pl/default-terms-of-service that would forward to a different URL

Reply:
Those permalinks are for in-product links and I'm not sure that they work with our docs set because IIRC it's a redirect that's set up in the wordpress console for the product/docs.

So in this case we'd use the link Carrie added and then if anything changes on our side we would set up a redirect on our side.